packages/ui-components/src/components/calendar-day/calendar-day.tsx

```diff
@@ -1,53 +1,53 @@
-import { Component, Event, EventEmitter, h, Host, Prop } from '@stencil/core';
-import { ICalendarDay, ICalendarDayEvents, IClickDayEvent, IMouseEnterEvent, IMouseLeaveEvent } from './calendar-day.types';
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+import { ICalendarDay, ICalendarDayEvents } from './calendar-day.types';
 
 /**
  * @part day-container - The day button container.
  */
 @Component({
 	tag: 'kv-calendar-day',
-	styleUrls: {
-		night: 'calendar-day.night.scss',
-		light: 'calendar-day.light.scss'
-	},
+	styleUrl: 'calendar-day.scss',
 	shadow: true
 })
 export class KvCalendarDay implements ICalendarDay, ICalendarDayEvents {
 	/** @inheritdoc */
-	@Prop({ reflect: true }) day: number;
+	@Prop({ reflect: true }) day!: number;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) disabled: boolean = false;
+	@Prop({ reflect: true }) disabled?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: true }) active: boolean = false;
+	@Prop({ reflect: true }) active?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: false }) inRange: boolean = false;
+	@Prop({ reflect: false }) inRange?: boolean = false;
 	/** @inheritdoc */
-	@Prop({ reflect: false }) leftRounded: boolean = false;
+	@Prop({ reflect: false }) isRangeStartDate?: boolean = false;
```

Comment:
IMO, the "start date" is not a part of this component and we came up with a different name for this prop.

Reply:
Here the notion of rounded does not apply because the styles for the selected day do not include border radius on the sides. Actually, after taking a better look, I think I dont need two props for this, I only need one like isDayStartOrEndOfRange.